lib/components/viewers/route-viewer.js

```diff
@@ -187,7 +187,9 @@ class RouteRow extends PureComponent {
   }
 
   render () {
-    const {isActive, ModeIcon, route, operator} = this.props
+    const {isActive, route, operator} = this.props
+    const { ModeIcon } = this.context
```

Comment:
Inconsistent spacing between braces.

Reply:
Fixed in 91bdbb5523830976b84ddf35078c2029899d5567.